nvidia-docker/README.md

```diff
@@ -1,7 +1,7 @@
 # NVIDIA Docker Examples
-*NVIDIA/ngc/nvidia-docker/README.md*
+*NVIDIA/ngc/nvidia-dockers/README.md*
 
-Examples of how [nvidia-docker][1] can be used to run GPU enabled jobs.
+Examples of how [nvidia-dockers][1] can be used to run GPU enabled jobs.
```

Comment:
Path was nvidia-docker not nvidia-dockers. I'd like to retain existing changes here.